src/core/data-cite.js

```diff
@@ -52,12 +52,17 @@ function requestLookup(conf) {
     if (frag) {
       href = new URL(frag, href).href;
     }
+    const cite = document.createElement("cite");
```

Comment:
Maybe keep this `createElement()` local to `case` where it is used?

Reply:
Sorry, I didn't get you. Actually I have used it inside `case: "a" and case "dfn"` block so if I define it there then I will have to define it twice.

Reply:
I was thinking we should define it as close to as it is used. Declaring twice shouldn't be an issue I think. WDYT @marcoscaceres

Reply:
Agree... would be fine to do:
 
```JS
        if (!path && !frag) {
           const cite = document.createElement("cite");
           elem.replaceWith(cite);
           cite.append(elem);
         }
```